src/System.Net.Security/tests/FunctionalTests/CertificateValidationClientServer.cs

```diff
@@ -21,11 +21,11 @@ public class CertificateValidationClientServer : IDisposable
 
         public CertificateValidationClientServer()
         {
-            _serverCertificateCollection = Configuration.Certificates.GetServerCertificateCollection();
-            _serverCertificate = Configuration.Certificates.GetServerCertificate();
+            _serverCertificateCollection = System.Net.Test.Common.Configuration.Certificates.GetServerCertificateCollection();
```

Comment:
+1 on the alias idea. 
Also if something worked before but doesn't now, it should be investigated. Please add a TODO/issue.